frontend/src/pages/Compare.tsx

```diff
@@ -332,12 +338,14 @@ class Compare extends Page<{}, CompareState> {
 
 const EnhancedCompare: React.FC<PageProps> = props => {
   const namespaceChanged = useNamespaceChangeEvent();
+  const { t, i18n } = useTranslation(['experiments', 'common']);
+  //const { t } = useTranslation(['experiments', 'common']);
```

Comment:
If the line is commented I only assume it was a test. Please remove it, or use the line appropriate. I think i18n is declared but not used